openapi/definitions.yaml

```diff
@@ -527,18 +527,18 @@ SenderMetadata:
     - service_name
     - organization_name
     - department_name
-Service:
-  description: A Service tied to an user's subscription.
+ServicePayload:
+  description: A payload used to create/update a service for a user.
```

Comment:
why remove service_id ? (we are used to create a service provinding its id)

Reply:
If needed you can use the Service type